e2e: Switch to testnet fixture (#1709)
ava-labs/avalanchego: tests/e2e/README.md

test run. When developing e2e tests, it may be helpful to create a
persistent test network to test against. This can increase the speed
of iteration by removing the requirement to start a new network for
every invocation of the test under development.

To use a persistent network:

```bash
# From the root of the avalanchego repo

# Build the testnetctl binary
$ ./scripts/build_testnetctl.sh

# Start a new network
$ ./build/testnetctl start-network --avalanchego-path=/path/to/avalanchego
...
Started network 1000 @ /home/me/.testnetctl/networks/1000

Configure testnetctl to target this network by default with one of the following statements:
 - source /home/me/.testnetctl/networks/1000/network.env
 - export TESTNETCTL_NETWORK_DIR=/home/me/.testnetctl/networks/1000
 - export TESTNETCTL_NETWORK_DIR=/home/me/.testnetctl/networks/latest

# Start a new test run using the persistent network
ginkgo -v ./tests/e2e -- \
    --avalanchego-path=/path/to/avalanchego \
    --ginkgo-focus-file=[name of file containing test] \
    --use-persistent-network \
    --network-dir=/path/to/network

# Also possible to set the AVALANCHEGO_PATH env var instead of supplying --avalanchego-path

> # Also possible to set the TESTNETCTL_NETWORK_DIR env var instead of supplying --network-dir

Reviewer: ```suggestion
# or TESTNETCTL_NETWORK_DIR instead of supplying --network-dir
```
